river/Cursor.zig

```diff
@@ -1095,3 +1099,44 @@ fn passthrough(self: *Self, time: u32) void {
         self.clearFocus();
     }
 }
+
+pub fn warp(self: *Self) void {
+    self.needWarp = false;
+    if (self.seat.focused_output == &server.root.noop_output) return;
+    // Warp pointer to center of the focused view/output (In layout coordinates) if enabled.
+    var layout_box: wlr.Box = undefined;
+    server.root.output_layout.getBox(self.seat.focused_output.wlr_output, &layout_box);
+    var box = switch (server.config.warp_cursor) {
```

Comment:
```suggestion
    const box = switch (server.config.warp_cursor) {
```
`target_box` might also be a bit more helpful as a name here.